tests/testsuite/build.rs

```diff
@@ -3726,14 +3808,13 @@ fn rustc_env_var() {
     p.cargo("build -v")
         .env("RUSTC", "rustc-that-does-not-exist")
         .with_status(101)
-        .with_stderr(
-            "\
-[ERROR] could not execute process `rustc-that-does-not-exist -vV` ([..])
+        .with_stderr_data(str![[r#"
+[ERROR] could not execute process `rustc-that-does-not-exist -vV` (never executed)
 
 Caused by:
-[..]
-",
-        )
+  [..]
```

Comment:
Platform dependent error:  

* No such file or directory (os error 2)
* program not found

Reply:
What are your thoughts on redacting the error?

Reply:
Fine with reaction, though finding the correct redactions would become a whac-a-mole game on Windows 😬.